scripts/run_e2e_tests_on_installed_operator.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -x
+
+
+E2E_TEST_DIR=./test/e2e
+ARGOCD_NS="openshift-gitops"
+DEPRACATED_ARGOCD_NS="openshift-pipelines-app-delivery"
+CONSOLE_LINK="argocd"
+
+APISERVER=https://kubernetes.default.svc
+
+# Path to ServiceAccount token
+SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+
+# Read this Pod's namespace
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+
+# Read the ServiceAccount bearer token
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+
+
+echo "
+apiVersion: v1
+kind: Config
+clusters:
+- name: default-cluster
+  cluster:
+    insecure-skip-tls-verify: true
+    server: ${APISERVER}
+contexts:
+- name: default-context
+  context:
+    cluster: default-cluster
+    namespace: ${NAMESPACE}
+    user: admin
+current-context: default-context
+users:
+- name: admin
+  user:
+    token: ${TOKEN}
+" > /tmp/output/sa.kubeconfig
+
+
+
+echo "Running e2e tests"
+CGO_ENABLED=0 SKIP_OPERATOR_DEPLOYMENT=true operator-sdk test local $E2E_TEST_DIR  --kubeconfig=/tmp/output/sa.kubeconfig --verbose --no-setup
```

Comment:
The local version of this would be:

* login to a cluster

Then,
 ```
E2E_TEST_DIR=./test/e2e CGO_ENABLED=0 SKIP_OPERATOR_DEPLOYMENT=true operator-sdk test local $E2E_TEST_DIR  --verbose --no-setup 
```

Reply:
It works, thanks

Created a make target - https://github.com/redhat-developer/gitops-operator/pull/82#discussion_r594936346